1819-1823MRMJournal.xml

```diff
@@ -9419,15 +9420,138 @@ Publisher:	London : [publisher not identified], 1786.
+                  <p>At home -- heard from dear Miss James with a present of a pretty new
+                     bonnet.</p>
+               </div>
+               <div type="entry" xml:id="e-959">
+                  <head><date when="1821-08-26">Sunday 26th</date></head>
+                  <p>Wrote to Miss James -- worked at prose.</p>
+               </div>
+               <div type="entry" xml:id="e-960">
+                  <head><date when="1821-08-27">Monday 27th</date></head>
+                  <p>At home -- worked a little at Lucy.</p>
+               </div>
+               <div type="entry" xml:id="e-961">
+                  <head><date when="1821-08-28">Tuesday 28th</date></head>
+                  <p>At home -- heard from Mr. Talfourd.</p>
+               </div>
+               <div type="entry" xml:id="e-962">
+                  <head><date when="1821-08-29">Wednesday 29th</date></head>
+                  <p>At home -- heard from Sir William -- finished my article on Lucy.</p>
+               </div>
+               <div type="entry" xml:id="e-963">
+                  <head><date when="1821-08-30">Thursday 30th</date></head>
+                  <p>At home -- heard from Sir William -- finished my article on Lucy.</p>
+               </div>
+               <div type="entry" xml:id="e-964">
+                  <head><date when="1821-08-31">Friday 31st</date></head>
+                  <p>At home -- heard delightfully from Mr. Talfourd -- went to Reading to make
+                     calls -- a pleasant morning.</p>
+               </div>
+               <div type="entry" xml:id="e-965">
+                  <head><date when="1821-09-01">Saturday 1st</date></head>
+                  <p>At home -- wrote to Miss James.</p>
+               </div>
+               <div type="entry" xml:id="e-966">
+                  <head><date when="1821-09-02">Sunday 2nd</date></head>
+                  <p>At home -- Mr. Crowther called -- Heard from Mrs. Jolliffe -- wrote to Miss
+                     James.</p>
+               </div>
+               <div type="entry" xml:id="e-967">
+                  <head><date when="1821-09-03">Monday 3rd</date></head>
+                  <p>At home -- worked at a prose article.</p>
+               </div>
+               <div type="entry" xml:id="e-968">
+                  <head><date when="1821-09-04">Tuesday 4th</date></head>
+                  <p>Went to Reading -- called at the Valpys &amp;c. -- pleasant morning -- home to
+                     dinner.</p>
+               </div>
+               <div type="entry" xml:id="e-969">
+                  <head><date when="1821-09-05">Wednesday 5th</date></head>
+                  <p>At home -- Mr. Crowther &amp; Miss Jephsojn called while we were out -- wrote
+                     to Mr. Crowther.</p>
+               </div>
+               <!-- nlh 2023-10-30:  the next 5 entries have [sic] next to their dates. what does it mean? -->
```

Comment:
@nhammer514 About this, she's listing the wrong day of the week! We can consult a cool piece of technology called a perpetual calendar to look up the year (1821) and find out what days of the week were for any month: https://www.timeanddate.com/calendar/?year=1821&country=1 

So in 1821, September 6 was on a Thursday...The question is, did she write the entry on Wed. 9/5, or Thurs. 9/6? Sometimes in the actual handwritten journal we see MRM go back and say something like "I've been in a mistake" about the dates...